bin/zopen-install

```diff
@@ -398,6 +399,32 @@ handlePackageInstall()
       printf "${NC}${HEADERCOLOR}${BOLD}${name}${NC}:\n ${installCaveat}\n" >> ${caveatsFile}
     fi
 
+    
+    systemPrereqs=$(jq -r '.product.system_prereqs // empty | map(.name) | join(" ")' "${metadataFile}" 2>/dev/null)
+    if ! $bypassPrereqs; then
+      if [ -z "${systemPrereqs}" ]; then
+        systemPrereqs="${systemPrereqs} zos24" # set the min requirest as zos24
+      fi
+      if [ -n "$systemPrereqs" ]; then
+        if [ ! -f "$ZOPEN_SYSTEM_PREREQ_SCRIPT" ]; then
+          printWarning "$ZOPEN_SYSTEM_PREREQ_SCRIPT does not exist. You should upgrade meta. Bypassing prereq check."
+        else
+          . $ZOPEN_SYSTEM_PREREQ_SCRIPT
+          for prereq in $(echo "${systemPrereqs}" | xargs | tr ' ' '\n' | sort -u); do
+            printInfo "Checking system pre-req requirement $prereq"
+            if command -V "${prereq}" >/dev/null 2>&1; then
+              if ! ( ${prereq} ); then
+                printError "Failed system pre-req check \"$prereq\". If you wish to bypass this, install with --bypass-prereq-checks"
+              fi
+            else
+              printWarning "$prereq does not exist in $ZOPEN_SYSTEM_PREREQ_SCRIPT. You should upgrade meta. Bypassing prereq check."
```

Comment:
We shouldn't bypass the prereq check if meta is in an odd state - either it's not updated enough to handle the specific prereq the package is trying to use or the package is broken and specifying an unknown prereq.